manta-crypto/src/algebra.rs

```diff
@@ -113,6 +141,100 @@ impl<G, const N: usize> PrecomputedBaseTable<G, N> {
     }
 }
 
+/// Window Method for Point Multiplication
+pub struct Window<G> {
+    /// Multiplication Table
+    table: Vec<G>,
+}
```

Comment:
```suggestion
/// Group Element Table for Windowed Point Multiplication
#[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq)]
pub struct Window<G> {
    /// Multiplication Table
    table: Vec<G>,
}
```

Reply:
Done, although it doesn't derive Copy (I guess because Vec<T> doesn't implement copy even when T does).